src/Core/Convert.php

```diff
@@ -557,22 +557,22 @@ public static function upperCamelToLowerCamel($str)
     {
         $return = null;
         $matches = null;
-        if (preg_match('/(^[A-Z]{1,})([A-Z]{1})([a-z]+.*)/', $str, $matches)) {
+        if (preg_match('/(^[A-Z]{1,})([A-Z]{1})([a-z]+.*)/', (string) $str, $matches)) {
```

Comment:
Retype as the first thing in the method and no need to worry about it in 3 different places afterwards?

Reply:
This was the largely the result of running rector/rector over the codebase, which will simply cast things to `(string)` to ensure they are not `null`.

This work looks to be bigger than originally envisioned (needs to be done across all of silverstripe/recipe-kitchen-sink, or not at all), so we're currently weighing up the priority of this against other competing priorities